pylab/core/models.py

```diff
@@ -0,0 +1,52 @@
+from autoslug import AutoSlugField
+from django_extensions.db.fields import CreationDateTimeField, ModificationDateTimeField
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext_lazy as _
+
+
+class Project(models.Model):
+    created = CreationDateTimeField()
+    modified = ModificationDateTimeField()
+    author = models.ForeignKey(User)
+    slug = AutoSlugField(populate_from='title')
+    title = models.CharField(_("Title"), max_length=255)
+    description = models.TextField(_("Description"))
+
+    def __str__(self):
+        return self.title
+
+    def get_absolute_url(self):
+        return reverse('project-details', args=[self.slug])
+
+
+class Event(models.Model):
+    OTHER = 0
+    PROJECT_DEVELOPMENT = 1
+    INITIAL_MEETING = 2
+    WEEKLY_MEETING = 3
+    EVENT_TYPE_CHOICES = (
```

Comment:
Currently event type is very static and should be hardcoded, because for some types of events, we do different things. So we have to have event type hardcoded to be able to us it in code. Also your examples also looks like an event of a particular type should be displayed differently or should behave in some different way. In order to be able to handle these differences in code, we have to define types in code.

Later, if we will see need, we can create separate model. But for now, I don't see why we would need that.